### PR TITLE
Stick with Ed25519VerificationKey2020 and EcdsaSecp256k1VerificationKey2019 but fix some text and examples (alt 1)

### PR DESCRIPTION
**Pull request by peacekeeper**
This is one out of 4 alternatives to address https://github.com/dhh1128/did-method-webs/issues/12.

The alternatives are:
- https://github.com/dhh1128/did-method-webs/pull/31 <-- this
- https://github.com/dhh1128/did-method-webs/pull/32
- https://github.com/dhh1128/did-method-webs/pull/33
- https://github.com/dhh1128/did-method-webs/pull/34

I suggest we use this PR here to discuss which one is best, and keep the others clean.

### CODE REVIEW

**Comment by dhh1128** [comment]
I don't really have a strong opinion. As a user of KERI, the CESR alternative would be really nice. It may be a bit cleaner and terser than the other alternatives, because it is self-describing (implying that new key types could get added to it without updating our spec, simply because CESR picks them up). Multikey has that same characteristic, I guess.

The obvious argument in favor of one of the others is: it's what other DID methods use. But before I endorse that argument, I'd like to know how strong it is. Specifically, which of these other methods is currently used by all or most did:web implementations, and does that method support multisig? If the answer is no, then the interop arg falls apart, doesn't it?

Bottom line is I'm more curious than opinionated on this topic.

**Comment by peacekeeper** [comment]
My impression is that the DID/VC community is more and more moving toward JsonWebKey and MultiKey. The Data Integrity spec defines something called a ["controller document"](https://w3c.github.io/vc-data-integrity/#controller-documents), which is basically like a DID document except that it potentially also works with identifiers other than DIDs. And they use JsonWebKey and MultiKey. I expect that a future W3C DID (maintenance) WG will also update the DID Core spec to align with this. So I think the interop argument is strong enough.

Note that there is a way to instruct a resolver to return the verification methods in a format that the client prefers, using the [transformKeys](https://github.com/decentralized-identity/did-spec-extensions/blob/main/parameters/transform-keys.md) extension DID parameter. So we could make the JsonWebKey type the default in the interest of interoperability, but a CESR-aware client could also request a CesrKey type.

Neither JsonWebKey nor MultiKey support multisig (despite the latter's name, haha!). For KERI multisig, I want to try to use [ConditionalProof](https://github.com/w3c-ccg/verifiable-conditions), see https://github.com/dhh1128/did-method-webs/issues/8. I will do a separate PR for that at some point.

**Comment by 2byrds** [comment]
Great points @dhh1128 and @peacekeeper! Multi-sig is a core feature, so whatever we choose it must-fully-support multi-sig.

**Comment by 2byrds** [comment]
Thoughts from our meeting today:

Sam noted:
Support for as many as possible is desired.
Should the default be the most expressive (CESRKey)?
Multi-sig is a core feature of KERI... it is 'multi-sig first'. So the default should be something that support multi-sig.
When it's single sig you could use the other representations.
We'd need to define how to list the keys in did:webs.

**Comment by peacekeeper** [comment]
Closing, since this was superseded by https://github.com/trustoverip/tswg-did-method-webs-specification/pull/39
